### PR TITLE
Updated Data - Experiment Details, Recommendation and Information for Templates. (Removed Dummy Data)

### PR DESCRIPTION
**Pull request by oumkale**
## Proposed changes

This PR has removed the dummy data from templates that were used for Experiment details.
Updated Files 
-Head
-Experiment
-Recommandation
-BrowserTemplate
-Redux Changes

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [x] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Review by ishangupta-ds — DISMISSED** [review]
LGTM.

**Review by amityt — APPROVED** [review]

**Comment by oumkale** [comment]
@rajdas98 PTAL

**Review by Jonsy13 — DISMISSED** [review]

File: litmus-portal/frontend/src/components/Sections/CreateWorkflow/ReliabilityScore/index.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import * as WorkflowActions from '../../../../redux/actions/workflow';
 import { RootState } from '../../../../redux/reducers';
 import ButtonFilled from '../../../Button/ButtonFilled';
 import ButtonOutline from '../../../Button/ButtonOutline';
-import InfoTooltip from '../../../InfoTooltip';
+// import InfoTooltip from '../../../InfoTooltip';
```
Comment (rajdas98) [inline litmus-portal/frontend/src/components/Sections/CreateWorkflow/ReliabilityScore/index.tsx:12]: remove the commented code

Comment (oumkale) [reply litmus-portal/frontend/src/components/Sections/CreateWorkflow/ReliabilityScore/index.tsx:12]: We need it in next release. St
ill should I?

Comment (rajdas98) [reply litmus-portal/frontend/src/components/Sections/CreateWorkflow/ReliabilityScore/index.tsx:12]: BCH will detect the commented codes. Currently, we don't have too much-commented code, that why it is not showing errors.
Selection_025

Comment (oumkale) [reply litmus-portal/frontend/src/components/Sections/CreateWorkflow/ReliabilityScore/index.tsx]: Done

**Review by S-ayanide — CHANGES_REQUESTED** [review]
This PR has a lot of extra interfaces and type changes which are not required. Adding the data in redux isn't required at all. I'll be making this change along with my PR its a minor change.

**Comment by S-ayanide** [comment]
@rajdas98 We can close this PR for now.

**Comment by oumkale** [comment]
@S-ayanide I already worked on this let me know changes. Why you want to close this PR. cc, @rajdas98 please keep this PR.

**Comment by oumkale** [comment]
Thank you for your review @S-ayanide but  I didn't add extra interfaces I just add a variable into it. Please Look carefully. cc, @rajdas98

File: litmus-portal/frontend/src/models/redux/workflow.ts [hunk 2]
```diff
@@ -11,6 +11,9 @@ export interface WorkflowData {
   weights: experimentMap[];
   isCustomWorkflow: boolean;
   clusterid: string;
+  details: string;
```
Comment (Jonsy13) [inline litmus-portal/frontend/src/models/redux/workflow.ts:14]: I don't think you need these in redux, we are not using these details,We are just showing them on frontend.
You can do that without redux.

File: litmus-portal/frontend/src/redux/actions/workflow.ts [hunk 3]
```diff
@@ -10,6 +10,9 @@ export const setWorkflowDetails = (data: {
   weights: experimentMap[];
   isCustomWorkflow: boolean;
   clusterid: string;
+  details: string;
```
Comment (Jonsy13) [inline litmus-portal/frontend/src/redux/actions/workflow.ts:13]: Not here as well.No need to store these details.

File: litmus-portal/frontend/src/redux/reducers/workflow.ts [hunk 4]
```diff
@@ -15,6 +15,9 @@ const initialState: WorkflowData = {
   weights: [],
   isCustomWorkflow: false,
   clusterid: '',
+  details: '',
```
Comment (Jonsy13) [inline litmus-portal/frontend/src/redux/reducers/workflow.ts:18]: Same

**Review by Jonsy13 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**